NuRadioReco/framework/base_station.py

```diff
@@ -322,3 +332,19 @@ def deserialize(self, data_pkl):
                 self.set_station_time(data['_station_time'])
 
         self._particle_type = data['_particle_type']
+
+
+    def __add__(self, x):
```

Comment:
doc string missing. Is it intuitive to implement such a operator? Maybe we should implement a `merge` or `combine` function

Reply:
missing doc string